src/Microsoft.Extensions.Logging/LogMessageFilter.cs

```diff
@@ -0,0 +1,4 @@
+namespace Microsoft.Extensions.Logging
+{
+    public delegate bool LogMessageFilter(string loggerType, string categoryName, LogLevel level);
```

Comment:
I'm thinking about removing it all together because most `AddFilter` overloads are forced to use Action<*> as filter delegates.